src/utils/openmc/summary.py

```diff
@@ -533,6 +532,14 @@ def _read_tallies(self):
 
 
     def make_opencg_geometry(self):
+
+        try:
+            from openmc.opencg_compatible import get_opencg_geometry
```

Comment:
PEP8 does say imports should all be at the top. That being said, OpenCG needs to remain as an optional dependency, so I'm ok with leaving this here for the time being.